src/ui.loader/index.vue

```diff
@@ -33,16 +33,25 @@ const props = defineProps({
     type: String,
     default: UIService.get(defaultConfig, ULoader).default.color,
   },
+
+  /**
+   * Make loader disabled.
+   */
+  disabled: {
```

Comment:
there should be `loading` prop not a `disabled`. Same as in other loaders. (and it should work by using v-if)
![image](https://github.com/vuelessjs/vueless/assets/16276298/c7d81b47-fb4a-46cc-bfab-dffc6645edb7)